webExtensionApis/study/schema.yaml

```diff
@@ -0,0 +1,164 @@
+# Interface schema
+# https://firefox-source-docs.mozilla.org/toolkit/components/extensions/webextensions/schema.html
+# possible types are here
+# - https://searchfox.org/mozilla-central/source/toolkit/components/extensions/Schemas.jsm#2418-2426
+#
+# - properties
+# - functions
+# - events
+# - types
+#
+# - type property:
+#   boolean, integer, number, string, array, object, or any.
+#
+
+# TODO
+# - add json schema types
+# - do we want to expose any of the sampling / hashing stuff?
+# - mention / decide how to handle all
+---
+- namespace: study
+  description: Interface for shield studies, for use by `background.js` scripts.
+  apiVersion: 5  # for dev use
+  types:
+  - name: studySetup
+  - name: weightedVariations
+  - name: anEnding
+  - name: telemetrySelectionOptions
+  - name: prefTypes
+
+  functions:
+  - name: configure
+    type: function
+    description: Configure the study.  Most things can't work without this
+    async: true
+    parameters:
+    - name: studySetup
```

Comment:
If this method is only called once for the lifetime of a study, we may want to consider adding a `duration` key (say, in weeks) to the `studySetup` parameter, and adding an `isExpired` method elsewhere in the API.